### PR TITLE
GSOC: add Keptn project ideas for GSoC 23

### PR DESCRIPTION
**Pull request by bradmccoydev**
Signed-off-by: Brad McCoy <bradmccoydev@gmail.com>

### CODE REVIEW

**Review by aliok — COMMENTED** [review]
Both mentors listed are in the [maintainers list](https://github.com/cncf/foundation/blob/main/project-maintainers.csv#L512)

File: summerofcode/2023.md [hunk 1]
```diff
@@ -187,3 +187,26 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
...
 - Upstream Issue (URL): https://github.com/WasmEdge/WasmEdge/issues/2244
+
+### Keptn
+
+#### Keptn Plugin for Backstage
+
+- Description: Backstage is an open platform for building developer portals. As the website states, "powered by a centralized software catalog, Backstage restores order to your infrastructure and enables your product teams to ship high-quality code quickly — without compromising autonomy". It would be nice to provide Keptn integration for this portal so that Keptn can connect as many as other tools. See all Backstage Plugins.  A Possible solution could be a specialized plugin for Keptn using Keptn REST API and/or CLI. It might be also possible to create a unified plugin for Keptn and other CI/CD tools, e.g. implemented on the top of the Cloud Events / CDEvents standard.
+- Expected outcome: Finding issues blocking multiple control planes, and possibly fixing them.
+- Recommended Skills: Typescript, Javascript, React, Backstage
+- Mentor(s): [Brad McCoy](https://github.com/bradmccoydev)
```
[the remark this answers is not in the document]
Comment (bradmccoydev) [reply summerofcode/2023.md:198]: Mentors: bradmccoydev@gmail.com and thomas.schuetz@t-sc.eu

[the remark this answers is not in the document]
Comment (aliok) [reply summerofcode/2023.md:198]: Sorry, I meant to ask you to add them to the doc in the PR. Like "Multiple Knative Eventing control planes" idea

**Comment by aliok** [comment]
LGTM

**Review by nate-double-u — APPROVED** [review]
Thanks for the proposal @bradmccoydev!

**Pull request merged**